src/suparnaNJune20/Assignment22/PalindromeString.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Assignment 22 : 1st Jul 2020
+
+
+Program 2:
+Verify given String is palindrome or not. 
+Input: radar
+Output : Given string radar is palindrome.
+
+Input: technocredits
+Output: Given string technocredits is not palindrome.
+ */
+package suparnaNJune20.Assignment22;
+
+public class PalindromeString {
+
+	void checkPalindromeStringMethod1(String str1) {
+		String tempStr = "";
+		char tempCh;
+		int index = 1;
+		while ((str1.length() - index) != -1) {
+			tempCh = str1.charAt(str1.length() - index);
+			tempStr = tempStr.concat(String.valueOf(tempCh));
+			index++;
+		}
+		if (str1.equalsIgnoreCase(tempStr))
+			// or if ((str1.compareToIgnoreCase(tempStr)) != 0)
+			System.out.println(str1 + " is Palindrome String  ");
+		else
+			System.out.println(str1 + " is not Palindrome String ");
+	}
+
+	void checkPalindromeStringMethod2(String str1) {
```

Comment:
Code looks good to me.

Reply:
Another method can be simply reverse the input string and then compare it with original one.
number of coding lines will also get decreased. you can try if you want. :)